docs/src/pages/components/buttons/buttons.md

```diff
@@ -180,3 +180,30 @@ However:
 ```
 
 This has the advantage of supporting any element, for instance, a link `<a>` element.
+
+## Unstyled button
```

Comment:
I think that we have a new opportunity to have an appealing demo https://deploy-preview-27600--material-ui.netlify.app/components/buttons/#unstyled-button

<img width="280" alt="Capture d’écran 2021-08-09 à 01 19 54" src="https://user-images.githubusercontent.com/3165635/128648595-5297858b-61a2-4206-b155-1905ae59de7a.png">

We could also maybe handle dark mode this time, we didn't in https://deploy-preview-27600--material-ui.netlify.app/components/switches/#unstyled-switches.

cc @danilo-leal

Reply:
I used the design from the rebranding (sans the font). As for the dark mode - I guess it would actually look the same, but I'm happy to change this if @danilo-leal has an idea.

Reply:
Oh OK, I raised it because the padding seemed off

Reply:
I made just two very small tweaks to both the _**[Customizing the root element](https://codesandbox.io/s/unstyledbuttonssimple-material-demo-forked-niqel?file=/demo.js)**_ and _**[Complex customization](https://codesandbox.io/s/unstyledbuttoncustom-material-demo-forked-hpi27?file=/demo.js)**_ demos. Let me know what you think. Minor stuff...

Reply:
Do you have any alternatives to Helvetica for those less fortunate who don't have it on their systems?

For the complex customization demo I'll try to come up with something slightly more complex.

Reply:
Any Sans Serif font would work, really! 😆 Just used in the Sandbox because it was rendering with a serif one.

Reply:
@danilo-leal could you take a look at the advanced example now: https://deploy-preview-27600--material-ui.netlify.app/components/buttons/#unstyled-component
Is this acceptable for you?

Reply:
Looking awesome, nice work! Made only two minor changes at the transition speed:

```
  & polygon {
    fill: transparent;
    transition: all 800ms ease; // 700 before
    pointer-events: none; 
  }
```

```
    & .bg {
      fill: ${alpha(
        theme.palette.primary.main,
        theme.palette.action.activatedOpacity
      )};
      transition: fill 300ms ease-out; // 200 before
    }
```

Reply:
Great, I included them, thanks!